x-pack/docs/en/security/authentication/built-in-users.asciidoc

```diff
@@ -77,24 +77,23 @@ password.
 
 You must set the passwords for all built-in users.
 
-The +elasticsearch-setup-passwords+ tool is the simplest method to set the
+The `elasticsearch-reset-passwords` tool is the simplest method to set the
 built-in users' passwords for the first time. It uses the `elastic` user's
 bootstrap password to run user management API requests. For example, you can run
-the command in an "interactive" mode, which prompts you to enter new passwords
+the command in an interactive mode, which prompts you to enter new passwords
 for the `elastic`, `kibana_system`, `logstash_system`, `beats_system`, `apm_system`,
 and `remote_monitoring_user` users:
 
 [source,shell]
 --------------------------------------------------
-bin/elasticsearch-setup-passwords interactive
+bin/elasticsearch-reset-passwords interactive
 --------------------------------------------------
```

Comment:
The usage of these commands are not the same. The interactive mode of the `reset` tool only works for a single user and you need specify it. I am thinking that we may want to leave these changes out of this PR because it needs to be largely rewritten for recommending the new `reset` tool.